econml/iv/dr/_dr.py

```diff
@@ -318,7 +318,7 @@ def predict(self, X=None):
         X = self._transform_X(X, fitting=False)
         return self._model_final.predict(X).reshape((-1,) + self.d_y + self.d_t)
 
-    def score(self, Y, T, X=None, W=None, Z=None, nuisances=None, sample_weight=None):
+    def score(self, Y, T, X=None, W=None, Z=None, nuisances=None, sample_weight=None, groups=None):
```

Comment:
groups=groups?

Reply:
This is the method definition, so groups=None is correct.